tensorrt_llm/llmapi/mgmn_worker_node.py

```diff
@@ -1,10 +1,16 @@
 #!/usr/bin/env python3
 import logging
 
+import torch
```

Comment:
@dcampora We don't require the changes in this file. It is a workaround on 0.19 and we have officail fixes in main branch. So, you can revert all changes in this file.

Reply:
Thanks, done.